recipes/gym-saturation/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "gym-saturation" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 72858ae490aea02515ba040760a80b6a1722bd353efb13daf6cf0c7660674b9a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - poetry           
+  run:
+    - python >=3.7
+
+test:
+  imports:
+    - gym_saturation
+  requires:
+    - pip
+    - pytest
+    - gym
+    - lark-parser
```

Comment:
It seems like these should be under `run` as they are required by the package: https://github.com/inpefess/gym-saturation/blob/f1e113b1bd798dea6de7f7ca10a3b6021d330d9f/pyproject.toml#L27-L28